examples/tv-casting-app/android/App/app/src/main/java/com/chip/casting/util/GlobalCastingConstants.java

```diff
@@ -6,5 +6,5 @@ public class GlobalCastingConstants {
   public static final int SetupPasscode = 20202021;
   public static final int Discriminator = 0xF00;
   public static final boolean ChipCastingSimplified =
-      false; // set this flag to true to demo simplified casting APIs
+      true; // set this flag to true to demo simplified casting APIs
```

Comment:
probably should change comment to describe what happens when setting to false (or remove this toggle if no longer needed)

Reply:
Updated comment. We'll keep this toggle around for a little bit longer, but now it will come preset to true.